conf/mysql5.hibernate.cfg.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.